src/libraries/System.Drawing.Common/src/System/Drawing/Bitmap.cs

```diff
@@ -53,6 +54,26 @@ public Bitmap(Stream stream) : this(stream, false)
         {
         }
 
+        public unsafe Bitmap(Stream stream!!, bool useIcm)
```

Comment:
Note that the `!!` is being removed with https://github.com/dotnet/runtime/pull/68178.

Please use `ArgumentNullException.ThrowIfNull(stream);` instead.

Reply:
I am working on this.

Reply:
Done.